src/main/java/com/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -1,4 +1,69 @@
 package com.novi.TechItEasy.controllers;
 
+import com.novi.TechItEasy.exceptions.RecordNotFoundException;
+import com.novi.TechItEasy.model.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/techItEasy")
 public class TelevisionController {
+    private List<Television> tv;// 1. declare the list
+
+    public TelevisionController() {
+        tv = new ArrayList<>();// 2. initialise the list
+        Television tele = new Television(); // instantiate the class Television
+        tele.setBrandName("onida");//setting the attributes to the instance of television
+        tele.setTeleType("LCD");
+        tv.add(tele); // 3. adding list to instance of television class
+    }
+
+    @GetMapping(value = "/televisions")
+    public ResponseEntity getTelevisions() {
+        return new ResponseEntity<>(tv, HttpStatus.OK);
+    }
+
+    @GetMapping("/televisions/{index}")
+    public ResponseEntity getTelevision(@PathVariable int index) {
+        if (index >= 0 && index < tv.size()) {
+            Television t = tv.get(index);
```

Comment:
1-letter variabele namen zou ik exclusief voor indexes in loops gebruiken, zoals `for(int i = 0; i < array.length; i++)`